modules/lvms-deleting-cloned-volumes-in-single-node-openshift.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_content-type: PROCEDURE
+[id="lvms-deleting-cloned-volumes-in-single-node-openshift_{context}"]
+= Deleting cloned volumes in {sno}
+
+You can delete cloned volumes.
+
+.Procedure
+
+* To delete the cloned volume, delete the cloned PVC by running the following command:
```

Comment:
This is minor but I don't think you _need_ a bullet here since its just one step for this procedure.